extensions/interactions/EndExploration/EndExploration.html

```diff
@@ -2,28 +2,27 @@
 
 <script type="text/ng-template" id="interaction/EndExploration">
   <div ng-if="recommendedExplorationIds.length > 0">
-    Other explorations you might enjoy:
+    <span ng-if="!collectionId">Other explorations you might enjoy:</span>
+    <span ng-if="collectionId">Continue the collection:</span>
     <div style="color: red;" ng-repeat="expId in invalidExpIds" ng-if="isInEditorPreviewMode">
       <strong>Error:</strong> no published exploration found with ID '<[expId]>'
     </div>
-    <div layout="row" layout-wrap>
-      <md-card class="oppia-end-exploration-recommendation-card" ng-repeat="expId in recommendedExplorationIds track by $index">
-        <a ng-href="/explore/<[expId]>" target="<[isIframed ? '_blank' : '_self']>" ng-if="recommendedExplorationSummaries[$index]">
-          <img class="oppia-end-exploration-recommendation-image" ng-src="<[recommendedExplorationSummaries[$index].thumbnail_image_url]>">
-          <div class="oppia-end-exploration-recommendation-title">
-            <[recommendedExplorationSummaries[$index].title]>
-          </div>
-        </a>
-      </md-card>
-    </div>
+    <oppia-end-exploration-recommended-explorations recommended-exploration-ids="recommendedExplorationIds" recommended-exploration-summaries="recommendedExplorationSummaries" collection-id="collectionId"></oppia-end-exploration-recommended-explorations>
   </div>
 
   <div ng-if="!isIframed">
-    <a href="/" ng-if="!isInEditorPreviewMode">
-      <md-button class="md-button-success">
-        Return to gallery
-      </md-button>
-    </a>
+    <div ng-if="!isInEditorPreviewMode">
+      <a ng-href="/collection/<[collectionId]>" ng-if="collectionId">
+        <md-button class="md-button-success">
+          Return to collection
```

Comment:
Having both 'Return to Collection' and 'Return to Gallery' next to each other doesn't look right. If you're in a collection, I would recommend just having 'Return to Collection'. They can always return to the gallery by leaving the collection.